docs/modules/ROOT/pages/integrations/logging.adoc

```diff
@@ -54,14 +54,14 @@ the `Pod` itself and generate the URLs for all the associated logs.
 ====
 
 - KubeArchive has no responsibility for sending the logs to the logging system.
-This is all configured elsewhere and outside of KubeArchive.
+This is all configured elsewhere and outside KubeArchive.
```

Comment:
I think "outside of" is correct here as it is being used as an exclusion.

Reply:
I did the change because my IDE told be that the `of` was redundant, but English isn't my first language so I'll apply your suggestion :)

Reply:
applied!